### DISCUSSION TITLE
E*Trade Supplemental Cost Basis + W2 RSU Income

### DISCUSSION THREAD

**Discussion by aedc123** [hickeng/financial#159] [Filing support]
I filled out the spreadsheet and entered everything but now I'm just second guessing! Two big things: 1)e*Trade posted a Stock Plan Transaction Supplement that supposedly accounts for ordinary income provided by the employer. It's an adjustment to cost basis that for me, results in a total loss. 2)My VMW W-2 from the last few years does have amounts in Box 14 with the "RSU" code, which seem to indicate those are vested RSU amounts that have been included in ordinary income. Is this accounted for in the calculation or am I just totally off?

**Comment by hickeng** [comment]
The eTrade supplement is, at least for me and many others, just not useful for 2023. I'm not sure how they got some of the numbers in mine and at best they may have considered the whole merger as "sold VMW for the cash portion, got AVGO for free"... I speculate that if you use those numbers that you'd have to use 0.00 as the cost basis for AVGO sales in the future.
That could be highly beneficial in terms of deferring tax to the future...but I'm highly unsure whether it's a valid approach! Otherwise I'd have taken it myself ;)

Regarding RSUs - they are taxed as regular income when you receive them and don't have an ordinary income element associated with the sale. Their tax basis is set to the market value they had on vest as that's when you paid tax on their value.

ESPP are not taxed at all on receipt and you're taxed for any discount when you sell - that's the ordinary income part.
The field you're looking for in Other Income in W2 for ESPP sales is ESPPD iirc.
